api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -47,87 +47,90 @@
 	@Autowired
 	private GenericCohortQueries genericCohorts;
 	
-	// Looks for patients with first drug pickup (encounter type 18=S.TARV:
-	// FARMACIA) before or on end date
+	@Autowired
+	private GenericCohortQueries genericCohortQueries;
+	
+	/**
+	 * @return Cohort of patients with first drug pickup (encounter type 18=S.TARV: FARMACIA) before
+	 *         or on end date
+	 */
 	@DocumentedDefinition(value = "patientWithFirstDrugPickupEncounter")
 	public CohortDefinition getPatientWithFirstDrugPickupEncounterBeforeOrOnEndDate() {
```

Comment:
since this method is only used internally, change the access modifier to **private**